app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -327,6 +327,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.default_api_port = "8443";
     } else if ($scope.emsCommonModel.emstype === 'vmware_cloud') {
       $scope.emsCommonModel.default_api_port = "443";
+      $scope.emsCommonModel.event_stream_selection = "none";
```

Comment:
@miha-plesko Can I request for one more change in the UI? (I fixed something similar for one other provider, and wanted to make sure we address that here as well)
Right here, can we add --
`$scope.emsCommonModel.amqp_security_protocol = 'non-ssl';`

The Validate button lights up even if no Security Protocol selection is made. So in order to avoid that, a default Security value here will help. The above change will ensure that the `Non-SSL` protocol is always selected for AMQP.